### DISCUSSION TITLE
saved keys plugin enabling

### DISCUSSION THREAD

**Discussion by musaleh17** [strongswan/strongswan#1507] [Q&A] [answered]
Hi,

In strongSwan developer doc was mentioned /configure script for enabling the specific plugin, but I didnt find it, 
do you know where I can find it? and is there any guide on how to enable the plugin and build with it?

**Comment by tobiasbrunner** [comment, marked as answer]
Please see the [installation docs](https://docs.strongswan.org/docs/5.9/install/install.html).
